### DISCUSSION TITLE
Edit/Add image Expression

### DISCUSSION THREAD

**Discussion by Zvek** [lllyasviel/Fooocus#2259] [Q&A]
Hello,

Could someone help me edit my image correctly? :) I want to take my original image and recreate it with exactly the same style, including:

 - A shocked face
 - An angry face
 - Happy ...

I've tried many things but I can't seem to achieve good results.

So if someone could provide methods to maintain the style of the image and simply edit different expressions in the same style, I would greatly appreciate it.
![prompt test expression](https://github.com/lllyasviel/Fooocus/assets/13851164/561f29be-4cc1-4383-bd4b-8105b124a65d)

(Maybe an easy feature like it could be added ? "expression swap") lol

In the same way, any tips to create head position with my image ?

Thx ! 
Regards

**Comment by lgwjames** [comment]
there is a model on https://civitai.com/models with wildcards for expressions
in the file is

:/
:3
:<
:d
:o
:p
:q
:|
afraid
amused
anger
angry
anxious
ashamed
bad
bashful
bored
cold
confident
confused
contempt
coy
crazy
curious
depressed
determined
disgust
disgusted
ecstatic
embarrassed
enraged
envious
excited
fear
flustered
frightened
frown
frustrated
furled brow
grimace
guilty
happiness
happy
hopeful
hot
hungry
hurt
indifferent
interested
jealous
joyful
lonely
loving
miserable
naughty_face
nervous
neutral
optimistic
pained
proud
puzzled
regretful
relieved
sad
sadness
scared
shocked
shy
sick
skeptical
sleepy
smile
smug
sorry
stubborn
stupid
surprise
surprised
suspicious
thirsty
thoughtful
tired
withdrawn
worried
